<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rPlayer-Controller</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="m_tabMain">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>control</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="m_lblStatus">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>27</width>
              <height>27</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_pbPlayPause">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>play/pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_pbVolDown">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_pbVolUp">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_pbQuitPlayer">
            <property name="text">
             <string>quit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QListWidget" name="m_lstTrack"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="m_pbBan">
              <property name="text">
               <string>ban</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_pbSkip">
              <property name="text">
               <string>skip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_pbLove">
              <property name="text">
               <string>love</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QPushButton" name="m_pbSend">
            <property name="text">
             <string>send..</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_text"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="m_lblText"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>info</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="m_label1">
            <property name="text">
             <string>host IP:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_lblIP">
            <property name="text">
             <string>unknown</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
